server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -249,24 +250,26 @@
   private int avgSeriesPointNumberThreshold = 500000;
 
   /**
-   * When merge point number reaches this, merge the vmfile to the tsfile.
+   * Work when tsfile_manage_strategy is level_strategy. When merge point number reaches this, merge
+   * the files to the last level.
    */
   private int mergeChunkPointNumberThreshold = 100000;
 
   /**
-   * Is vm merge enable
+   * TsFile manage strategy, define use which hot compaction strategy
    */
-  private boolean enableVm = true;
+  private TsFileManagementStrategy tsFileManagementStrategy = TsFileManagementStrategy.LevelStrategy;
```

Comment:
Is `tsFileManagementStrategy` a config which could only be modified at the first time starting IoTDB? If so, you may have to add it to `IoTDBConfigCheck` to abandon changing it when restarting.